src/assets/usage-examples-code/sprites/draw_sprite/draw_sprite-2-draw-sprite.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * Usage Example: drawing a sprite
+ **/
+
+#include "splashkit.h"
+
+int main()
+{
+    // window to draw the sprite on
+    window start = open_window("create_sprite", 600, 600);
+
+    // bitmap for creating a sprite
+    bitmap player = load_bitmap("playerBmp", "protSpriteSheet.png");
```

Comment:
The "protSpriteSheet.png" file is missing from the PR; please include it.

Reply:
Added the file